aten/src/ATen/native/NNPACK.cpp

```diff
@@ -121,7 +121,11 @@ struct Workspace {
     constexpr size_t nnpack_memory_alignment_boundary = 64;
 
     // Won't work on Windows, but NNPACK doesn't support Windows either
-    posix_memalign(&buffer, nnpack_memory_alignment_boundary, size);
+    auto res = posix_memalign(&buffer, nnpack_memory_alignment_boundary, size);
```

Comment:
Since we are on C++17, can we use std::aligned_alloc now?

Reply:
I will try on another PR